include/mesh.tcc

```diff
@@ -1028,44 +1098,78 @@ bool mpm::Mesh<Tdim>::read_particles_hdf5(unsigned phase,
-      HOFFSET(HDF5Particle, nsize_y),    HOFFSET(HDF5Particle, nsize_z),
-      HOFFSET(HDF5Particle, velocity_x), HOFFSET(HDF5Particle, velocity_y),
-      HOFFSET(HDF5Particle, velocity_z), HOFFSET(HDF5Particle, stress_xx),
-      HOFFSET(HDF5Particle, stress_yy),  HOFFSET(HDF5Particle, stress_zz),
-      HOFFSET(HDF5Particle, tau_xy),     HOFFSET(HDF5Particle, tau_yz),
-      HOFFSET(HDF5Particle, tau_xz),     HOFFSET(HDF5Particle, strain_xx),
-      HOFFSET(HDF5Particle, strain_yy),  HOFFSET(HDF5Particle, strain_zz),
-      HOFFSET(HDF5Particle, gamma_xy),   HOFFSET(HDF5Particle, gamma_yz),
-      HOFFSET(HDF5Particle, gamma_xz),   HOFFSET(HDF5Particle, epsilon_v),
-      HOFFSET(HDF5Particle, status),     HOFFSET(HDF5Particle, cell_id),
+      HOFFSET(HDF5Particle, id),
+      HOFFSET(HDF5Particle, mass),
+      HOFFSET(HDF5Particle, volume),
+      HOFFSET(HDF5Particle, pressure),
+      HOFFSET(HDF5Particle, coord_x),
+      HOFFSET(HDF5Particle, coord_y),
+      HOFFSET(HDF5Particle, coord_z),
+      HOFFSET(HDF5Particle, displacement_x),
+      HOFFSET(HDF5Particle, displacement_y),
+      HOFFSET(HDF5Particle, displacement_z),
+      HOFFSET(HDF5Particle, nsize_x),
+      HOFFSET(HDF5Particle, nsize_y),
+      HOFFSET(HDF5Particle, nsize_z),
+      HOFFSET(HDF5Particle, velocity_x),
+      HOFFSET(HDF5Particle, velocity_y),
+      HOFFSET(HDF5Particle, velocity_z),
+      HOFFSET(HDF5Particle, stress_xx),
+      HOFFSET(HDF5Particle, stress_yy),
+      HOFFSET(HDF5Particle, stress_zz),
+      HOFFSET(HDF5Particle, tau_xy),
+      HOFFSET(HDF5Particle, tau_yz),
+      HOFFSET(HDF5Particle, tau_xz),
+      HOFFSET(HDF5Particle, strain_xx),
+      HOFFSET(HDF5Particle, strain_yy),
+      HOFFSET(HDF5Particle, strain_zz),
+      HOFFSET(HDF5Particle, gamma_xy),
+      HOFFSET(HDF5Particle, gamma_yz),
+      HOFFSET(HDF5Particle, gamma_xz),
+      HOFFSET(HDF5Particle, epsilon_v),
+      HOFFSET(HDF5Particle, status),
+      HOFFSET(HDF5Particle, cell_id),
   };
 
   // To get size
   HDF5Particle particle;
 
   size_t dst_sizes[NFIELDS] = {
-      sizeof(particle.id),         sizeof(particle.mass),
-      sizeof(particle.volume),     sizeof(particle.pressure),
-      sizeof(particle.coord_x),    sizeof(particle.coord_y),
-      sizeof(particle.coord_z),    sizeof(particle.nsize_x),
-      sizeof(particle.nsize_y),    sizeof(particle.nsize_z),
-      sizeof(particle.velocity_x), sizeof(particle.velocity_y),
-      sizeof(particle.velocity_z), sizeof(particle.stress_xx),
-      sizeof(particle.stress_yy),  sizeof(particle.stress_zz),
-      sizeof(particle.tau_xy),     sizeof(particle.tau_yz),
-      sizeof(particle.tau_xz),     sizeof(particle.strain_xx),
-      sizeof(particle.strain_yy),  sizeof(particle.strain_zz),
-      sizeof(particle.gamma_xy),   sizeof(particle.gamma_yz),
-      sizeof(particle.gamma_xz),   sizeof(particle.epsilon_v),
-      sizeof(particle.status),     sizeof(particle.cell_id),
+      sizeof(particle.id),
```

Comment:
Could you reformat to as it was like before?

Reply:
It is done automatically by `clang`. We can revert back maybe by changing from `displacement` to `disp`. Do you want me to do that?

Reply:
OK leave it as is